host-macro/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "trouble-host-macro"
```

Comment:
Maybe name this 'trouble-host-macros', as I think it can be used for more than the derive macros.

Reply:
Renamed in 87b7dae6a8ea9bd02212610476f57c731c7b179b